azurerm/internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -61,6 +66,84 @@ func resourceMsSqlVirtualMachine() *schema.Resource {
 				}, false),
 			},
 
+			"auto_backup": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"backup_system_databases": {
```

Comment:
should we put enabled on this?
```suggestion
						"backup_system_databases_enabled": {
```

Reply:
Sure, how about `system_databases_backup_enabled`?

Reply:
Or rather, since it defaults true, perhaps reversing it to `manual_backup_schedule_enabled` would be more fitting